drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -59,7 +60,16 @@
 INCOMING_DATE_FORMAT = '%d.%m.%Y'
 OPENBIS_DATE_FORMAT = '%Y-%m-%d'
 
-PROPPERTY_FILTER_LIST = ["IMAGE_FILENAME", "INSTRUMENT_USER", "IMAGING_DATE"]
+#PROPPERTY_FILTER_LIST = ["IMAGE_FILENAME", "INSTRUMENT_USER", "IMAGING_DATE"]
+PROPPERTY_FILTER_LIST = ["IMAGE_FOLDER_PATH", "INSTRUMENT_USER", "IMAGING_DATE", "SAMPLE_ID", "OMERO_TAGS", "ETL_TAG"]
+
+def log_print(msg_string):
+	import time
+
+	dt_string = time.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Why not use the OpenBis DateFormat Variable here? 
```suggestion
	dt_string = time.strftime('%Y-%m-%d %H:%M:%S')
```

Reply:
This is just a format to make the logging clearer, not sure if we need to stay consistent with the OpenBis DateFormat Variable